pyigm/fN/mockforest.py

```diff
@@ -52,56 +52,60 @@ def dopp_val(x,bsig=24*u.km/u.s,bmnx=(15.,80)*u.km/u.s):
     return bx
 
 
-def monte_HIcomp( zmnx, fN_model, NHI_mnx=None, dz=0.001, cosmo=None,
+def monte_HIcomp(zmnx, fN_model, NHI_mn=None, bfix=None, dz=0.001, cosmo=None,
     rstate=None, seed=None):
     """ Generate a Monte Carlo draw of HI components (z,N,b)
-
+        
     Parameters
-    ----------
+     ----------
     zmnx : tuple (float,float)
-      Redshift range for analysis.
-      Should correspond to Lya
+        Redshift range for analysis.
+        Should correspond to Lya
     fN_model : fN_Model class
     NHI_mnx : tuple, optional (float,float)
-      Range of logNHI for linelist
+        Range of logNHI for linelist
+    bfix : float, optional
+        None for using random b, float for a fixed b
     dz : float, optional
-      Step size for z discretization
+        Step size for z discretization
     cosmo : astropy Cosmology, optional
     rstate : RandomState, optional
-      For random number generation
+        For random number generation
     seed : int, optional
-      Seed for random numbers
-
+        Seed for random numbers
+        
```

Comment:
Please make sure you are not adding "Tabs"
to the file.  Indentation should only be with spaces.  No Tabs.